data/ecosystems/c/celo.toml

```diff
@@ -14,7 +14,9 @@ sub_ecosystems = [
   "Axelar Network",
   "Band Protocol",
   "Beefy.Finance",
+  "Bidali",
   "Blockcash Finance",
+  "BlockDaemon",
```

Comment:
we keep DefiLlama as general with no sub-ecos since it covers all crypto

Reply:
DefiLlama or BlockDaemon?

Reply:
Removed